### DISCUSSION TITLE
Change artifacts expiration

### DISCUSSION THREAD

**Discussion by Seg9** [codemagic-ci-cd/codemagic-docs#2524] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [x] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Android (Kotlin)

### Steps to reproduce

Hi, I would like to ask you, if it is possible to change expiration date for artifacts. As I know, artifacts are stored for 60 days by default with Pay as you go plan. But would it be possible to limit the storage time, for example to 30 days, if we don't want to store it for such a long time? Thank you for your response/

### Expected results

-

### Actual results

-

**Comment by dtrdic** [comment, marked as answer]
hey @Seg9 you can adjust the expiration date of the download link for the binary by navigating to Teams > team name > artifact download link. While it can't be set for 30 days, you can set it for one week. Hope this helps

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/b000e85d-9061-4fb0-bbe2-2b8922bffd84)

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
